client/serviceinterface_update.go

```diff
@@ -197,12 +197,12 @@ func (cli *VanClient) ServiceInterfaceBind(ctx context.Context, service *types.S
 		if err != nil {
 			return err
 		}
-		err = validateCrossNamespacePermissions(cli, namespace)
+		_, svcNamespace := kube.GetServiceName(targetName, utils.GetOrDefault(namespace, cli.GetNamespace()))
```

Comment:
I don't understand why we want this change. We only need to check for the extra permissions if a different namespace has been specified. `err = validateCrossNamespacePermissions(cli, namespace)` seems the right way to do that to me. Or am I missing something?

Reply:
If the target is a qualified service name, then the target namespace is part of the targetName,
so we pick the namespace from it, otherwise we use the namespace provided.

Reply:
But we don't need to do any permission validation in that case.

Reply:
I don't think we should try to read the service unless the target name is a service in the site namespace. That is what I believe we did before the introduction of the target namespace.

Reply:
I will make it this change for your evaluation. One thing that will happen is that, once ports cannot be
deduced, user will need to provide `--port` when exposing a service from a different namespace. Which
seems reasonable to me. I will push it then let me know if you're ok with it.